### DISCUSSION TITLE
Unverifiable Proof Certificate of Studentship

### DISCUSSION THREAD

**Discussion by wraith4081** [community/community#45739] [GitHub Education]
Hello everyone, I am a high school student. I have been trying to renew my Github Student membership for a while. Since high school level schools do not have a student certificate to verify my student status, I got a student certificate approved by the Ministry of National Education, the official transaction management panel in my country. But when I try to add it as a student ID, it does not accept it. And in other options, it says that I need to upload a student ID. How can I solve this problem?

**Comment by jge162** [comment]
Hello @wraith4081 in your particular scenario I would recommend you reach out to [GitHub Education](https://support.github.com/request/education?tags=hubberfy_education) support to better assist you as we cannot make determinations (we are just users in a discussion) -> here. Thank you for you posts. `Choose the option you feel bests suits you on support page.`
<img width="1091" alt="Screenshot 2023-01-29 at 6 32 35 PM" src="https://user-images.githubusercontent.com/31228460/215375286-7483a14b-d1b8-42dd-acd9-5dd0f1443ecf.png">
